Update check_test docstring to clarify state parameter

mikigo opened the pull request:
In https://github.com/avocado-framework/avocado/blob/3b1f5cbbb07ea850498b55aa020d74cc0f7fa15f/avocado/core/result.py#L103

The parameter description of 'test' does not match the parameter 'state'

and

Refactor: Remove unused 'option = None' initialization

Signed-off-by: mikigo <huangmingqiang@uniontech.com>

richtja reviewed (changes requested):
Hi @mikigo, thank you for your updates. I think there was a small misunderstanding on #6052, sorry about that. What I meant was that you need to add `Signed-off-by: mikigo <huangmingqiang@uniontech.com>` to your commits messages no to PRs message, can you please fix that. Also, on  #6052 your changes were split into two commits, which is a right choice. Please do that again. 

Small tip, you don't need to close this PR and open a new one. You can apply your changes with `git push --force` to this PR.

mikigo commented:
Thank you for your feedback! I have made the necessary changes as requested.

- Modified Commit Messages: I have updated the commit messages to include the required information.

- Split Commits: I have split the previous single commit into two separate commits.

- Force Pushed: I have force pushed the changes to this PR.

Please review the changes and let me know if there is anything else I need to address.

Thanks again for your guidance!

Best regards,
mikigo

richtja reviewed (approved):
Hi @mikigo, thank you for the updates it LGTM, the CI failures are not related to this PR.
